lib/v8debugapi.js

```diff
@@ -385,9 +385,28 @@ module.exports.create = function(logger_, config_, fileStats_) {
 
   function findScripts(scriptPath) {
     var regexp = pathToRegExp(scriptPath);
-    var matches = Object.keys(fileStats).filter(function(script) {
+    var filterScripts = function(script) {
       return regexp.test(script);
```

Comment:
The closure-captured `regexp` is mutable. It would be better to pass it as an argument, or possibly, just remove filterScripts completely just bind and pass `regexp.test` to `Array.prototype.filter`.